README.md

```diff
@@ -1,7 +1,14 @@
 # AMP HTML ⚡
 
-[![Renovate enabled](https://img.shields.io/badge/renovate-enabled-brightgreen.svg)](https://renovateapp.com/)
-[![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
+[![GitHub Release](https://img.shields.io/github/release/ampproject/amphtml.svg)](https://github.com/ampproject/amphtml/releases)
+[![Contributors](https://img.shields.io/github/contributors-anon/ampproject/amphtml.svg)](https://github.com/ampproject/amphtml/graphs/contributors)
+[![Build Status](https://img.shields.io/travis/ampproject/amphtml/master.svg)](https://travis-ci.org/ampproject/amphtml)
+[![GitHub Issues](https://img.shields.io/github/issues/ampproject/amphtml.svg)](https://github.com/ampproject/amphtml/issues)
+[![GitHub Pull Requests](https://img.shields.io/github/issues-pr-closed/ampproject/amphtml.svg)](https://github.com/ampproject/amphtml/pulls)
```

Comment:
Small thing, but why "open issues" but "closed PRs"?

Reply:
I figured closed PRs reflect what we've done, and open issues reflect what we have left to do :smiley: 

There are more options listed here: https://shields.io/category/issue-tracking. Let me know if you prefer other metrics.

Reply:
Updated with both open and closed issues / PRs.